test/tools/evmoneRunner.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <test/tools/evmoneRunner.h>
+#include <iostream>
+#include <assert.h>
+#include <fstream>
+#include <limits>
+
+using namespace external::evmone;
+using namespace std;
+
+evmc_host_interface EvmOneVM::interface{
+		{},
+		{},
+		{},
+		{},
+		{},
+		{},
+		{},
+		{},
+		{},
+		[](evmc_context* ctx) { return static_cast<EvmOneVM*>(ctx)->tx_context; },
+		{},
+		{},
+};
+
+void EvmOneVM::execute(const evmc_message& _m, std::basic_string<uint8_t> _code)
+{
+	evmc_result result = {};
+	result = vm->execute(vm, this, rev, &_m, &_code[0], _code.size());
+	assert(result.status_code == EVMC_SUCCESS);
+	// Checking if execution output is true or 1
+	//	assert(result.output_data[31] == '\x01');
```

Comment:
The commented `assert` checks if the EVM bytecode returns true or not

Reply:
Is it going to be uncommented before merging?